log_parser.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo -n "{}" > filter_logs.json
+declare -a filters=("status" "retention_schedule" "litigation_hold" "contact_phone" "language" "referred" "hate_crime" "servicemember" "intake_format" "location_state" "primary_complaint" "contact_first_name" "contact_last_name" "contact_email" "violation_summary" "location_name" "location_name_1" "location_name_2" "location_city_town" "create_date_start" "create_date_end" "summary" "origination_utm_campaign" "assigned_to" "dj_number" "tags" "public_id" "primary_statute" "district" "reported_reason" "commercial_or_public_place")
```

Comment:
```suggestion
declare -a filters=(\
  "assigned_to"\
  "commercial_or_public_place"\
  "contact_email"\
  "contact_first_name"\
  "contact_last_name"\
  "contact_phone"\
  "create_date_end"\
  "create_date_start"\
  "district"\
  "dj_number"\
  "hate_crime"\
  "intake_format"\
  "language"\
  "litigation_hold"\
  "location_city_town"\
  "location_name"\
  "location_name_1"\
  "location_name_2"\
  "location_state"\
  "origination_utm_campaign"\
  "primary_complaint"\
  "primary_statute"\
  "public_id"\
  "referred"\
  "reported_reason"\
  "retention_schedule"\
  "servicemember"\
  "status"\
  "summary"\
  "tags"\
  "violation_summary"\
)
```

Putting this on multiple lines might make it easier to read / edit